### PR TITLE
Fix new_rate var not declared

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: changes/tests.py [hunk 1]
```diff
@@ -547,6 +547,100 @@ def test_prebirth_text_to_audio_week28_new_short_name(self):
...
+            responses.GET,
+            'http://localhost:8005/api/v1/messageset/%s' % query_string,
+            json={
+                "count": 1,
+                "next": None,
+                "previous": None,
+                "results": [{
+                    "id": 4,
+                    "short_name": 'prebirth.mother.text.0_9',
+                    "default_schedule": 1
```
Comment (KaitCrawford) [inline changes/tests.py:624]: Shouldn't the new subscription and the old subscription be on different schedules if you want to test the new `next_sequence_number`?

**Review by KaitCrawford — COMMENTED** [review]
Left a comment

File: changes/tests.py (hunk 1 above)
Comment (erikh360) [reply changes/tests.py:624]: I'm simulating a text to text change which is on the same schedule, the test above this one changes to a different schedule.

Comment (KaitCrawford) [reply changes/tests.py:624]: ah ok

**Review by KaitCrawford — APPROVED** [review]
:+1:

**Pull request merged**